### ISSUE TITLE
trainin loss

### ISSUE THREAD

**Issue by OliverLeeXZ** [tykim0507/Motion-LoRA#4]
I using lora to finetune SVD, the num of training samples is 53, however, looking at the loss during training, it seems that the training has crashed. Could you tell me the reason why this problem occur and how to solve this problem?  

![training_loss](https://github.com/user-attachments/assets/d3c7c5d6-7a37-4eff-9cf0-6f3962c6acd1)
